### DISCUSSION TITLE
Using other DBs like Surrealdb or Neo4j for socializing

### DISCUSSION THREAD

**Discussion by axscell** [supabase/supabase#18266] [Feature Requests]
I was wondering why the core DB is not using other DBs like Neo4j or surrealdb in it's core. This might increase the speed of development and be convenient for large apps or other purpose apps like social networking. What is wrong with implementing in other DBs? Is there any discussion?

**Comment by notramo** [comment]
SurrealDB and EdgeDB already have builtin nice UI, vector store, HTTP & GraphQL API built-in. EdgeDB also has Auth and geospatial data support. SurrealDB has realtime support.

This is nearly the entire feature set of Supabase. These databases include the features that Supabase adds to Postgres. We could say Supabase modernizes Postgres to get to the DX of newer databases.
